README.md

```diff
@@ -5,9 +5,9 @@ https://inlong.apache.org/
 
 ## Prerequisite
 
-InLong website is powered by [docsite](https://docusaurus.io/).
-If your version of docsite is less than `2.0.0`,please upgrade to `2.0.0`.
-Please also make sure your node version is 14.x, versions higher than 14.x is not supported by docsite yet.
+InLong website is powered by [docusaurus](https://docusaurus.io/).
```

Comment:
Should the first letter of "docusaurus" be capitalized?
I understand that this modification is to mark the name of our front-end component. If it is all lowercase, I find that it cannot be translated

Reply:
done